agents/holo_synth/synth_agent.py

```diff
@@ -0,0 +1,208 @@
+import argparse
+import os
+import time
+
+import numpy as np
+import txaio
+import yaml
+
+ON_RTD = os.environ.get("READTHEDOCS") == "True"
+if not ON_RTD:
+    from holog_daq import synth3
+    from ocs import ocs_agent, site_config
+    from ocs.ocs_twisted import Pacemaker, TimeoutLock
+
+
+class SynthAgent:
+    """
+    Agent for connecting to the Synths for holography
+    """
+
+    def __init__(self, agent, config_file):
+
+        self.lo_id = None
+        self.initialized = False
+        self.take_data = False
+
+        self.agent = agent
+        self.log = agent.log
+        self.lock = TimeoutLock()
+
+        agg_params = {"frame_length": 10 * 60}  # [sec]
+        self.agent.register_feed(
+            "synth_LO", record=True, agg_params=agg_params, buffer_time=0
```

Comment:
Feed names need to be all lowercase. They can also contain numbers and underscore. So "synth_lo" is alright here. Note you'll need to change the name in the publish call as well.

Reply:
Feed name still needs to be changed on lines 34 and 139 from "synth_LO" to "synth_lo".

Reply:
Ok in synth_agent.py, I've replaced all instances of synth_LO to synth_lo

Reply:
I've also changed to synth_lo in the field.yaml file, let me know if im forgetting anywhere else to update this.